src/modinfoforeign.cpp

```diff
@@ -49,7 +42,15 @@ ModInfoForeign::ModInfoForeign(const QString& modName, const QString& referenceF
     : ModInfoWithConflictInfo(core), m_ReferenceFile(referenceFile),
       m_Archives(archives), m_ModType(modType)
 {
-  m_CreationTime = QFileInfo(referenceFile).birthTime();
+  m_CreationTime          = QFileInfo(referenceFile).birthTime();
+  IPluginGame const* game = qApp->property("managed_game").value<IPluginGame*>();
+  QMap<QString, QDir> directories = {{"data", game->dataDirectory()}};
```

Comment:
`"data"` should probably be fetch from `dataDirectory`, typically for Morrowind it's `Data Files`.

Reply:
This is more of an arbitrary string so that I can merge the maps. It actually corresponds to the internal origin 'name' as used by MO2 (which is, in fact, 'data'). But it doesn't correspond with the actual directory name.